src/library_pthread.js

```diff
@@ -440,34 +430,49 @@ var LibraryPThread = {
     // Creates a new web Worker and places it in the unused worker pool to wait for its use.
     allocateUnusedWorker() {
       var worker;
+      var workerOptions = {
+        {{{ EXPORT_ES6 ? "type: 'module'," : "" }}}
```

Comment:
I think this will be clearer:
```suggestion
#if EXPORT_ES6
        'type': 'module',
#endif
```
(I also quoted the key to prevent Closure compiler from renaming/mangling the property)